docs/.vitepress/config.ts

```diff
@@ -86,6 +86,7 @@ function sidebarGuide() {
       text: 'Writing',
       collapsible: true,
       items: [
+        { text: 'Inline highlight', link: '/guide/inline' },
```

Comment:
I don't think it is necessary to add an md file to explain
Can add headings in `docs/guide/markdown.md` in `Import Code Snippets` and `Markdown File Inclusion`

Reply:
sure. it's just to show the function for review.

Reply:
Well. Space split(or replace) possible is something wrong

```
- `const a: string = 'hello world'`{ts}
```

<img width="698" alt="image" src="https://user-images.githubusercontent.com/40693636/191400513-71097683-1263-4881-9634-48cd4c7a82e7.png">

Reply:
Fixed. PTAL https://deploy-preview-1366--vitepress-docs.netlify.app/guide/inline